src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -537,6 +537,32 @@ def prepare_latents(self, batch_size, num_channels_latents, height, width, dtype
         latents = latents * self.scheduler.init_noise_sigma
         return latents
 
+    def enable_freeu(self, s1: float, s2: float, b1: float, b2: float):
```

Comment:
Is it possible set some reasonable defaults for these parameters? e.g. Like the ones provided in the test.

Reply:
It's different for SD v1.5, different for SD v2.1, and for SDXL. So, not sensible IMO. @patrickvonplaten and @apolinario also alluded to the same thing.